routing_model/Cargo.toml

```diff
@@ -1,11 +1,15 @@
 [package]
+authors = ["MaidSafe Developers <dev@maidsafe.net>"]
+description = "A secured storage DHT"
```

Comment:
is that right?

Reply:
Nope - I wrote a description and have obviously obliterated it with the Routing one :D  I'll fix this.